packages/react-components/src/utils/setupConfiguration.ts

```diff
@@ -289,19 +311,57 @@ async function fetchVersionName(defaultValue: string): Promise<string> {
     return getConfigurationStanzaValue(service, 'app', 'launcher', 'version', defaultValue);
 }
 
+function convertSeverityFilterToArray(
+    severitiesFilter: string[],
+    allSeverities: Severity[]
+): Severity[] {
+    const severities: Severity[] = [];
+    severitiesFilter.forEach((severityValue) => {
+        if (severityValue === DEFAULT_FILTER_VALUE) {
+            severities.push(...allSeverities);
+        } else {
+            const foundSeverityMatch = allSeverities.find(
+                (severity) => severity.value === severityValue
+            );
+            if (foundSeverityMatch) {
+                severities.push(foundSeverityMatch);
+            }
```

Comment:
Not sure to get this one, doing a for loop to find the severity is basically the same as doing a find(), right? Was the problem more regarding the first condition for the default filter value? In the new patchset, I removed this altogether since it was old code to support an empty filter

Reply:
For me, it's about fewer lines of code and no need for the variable `foundSeverityMatch`.

Reply:
In your latest change, again some contextual spacing goes a long way in terms of readability. Blobbing everything together with no line spacing increases cognitive load.

Reply:
I will separate the code more and add some comments for clarity. The reason I prefer using a find() instead of a forEach is because you can't leave early in a forEach unless you throw an Exception. Would be nice if you could push only if the variable wasn't undefined